migration/migration.go

```diff
@@ -0,0 +1,56 @@
+package migration
+
+import (
+	"fmt"
+	"os"
+	"practice/go-todo-list/config"
+
+	"github.com/golang-migrate/migrate"
+	_ "github.com/golang-migrate/migrate/database/postgres"
+	_ "github.com/golang-migrate/migrate/source/file"
+	_ "github.com/golang-migrate/migrate/source/github"
+	_ "github.com/lib/pq"
+)
+
+var migrationPath = "file://" + os.Getenv("GOPATH") + "/src/practice/go-todo-list/migration/queries/"
+
+var runner *migrate.Migrate
+
+func Init() {
+	connURL := config.Db().ConnString()
+	var err error
+	fmt.Println(migrationPath, connURL)
```

Comment:
print statement again. please delete this

Reply:
Done.